src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenOnBehalfOfParameterBuilder.cs

```diff
@@ -181,7 +181,7 @@ protected override void Validate()
         /// <inheritdoc />
         internal override ApiEvent.ApiIds CalculateApiEventId()
         {
-            return ApiEvent.ApiIds.AcquireTokenOnBehalfOf;
+            return string.IsNullOrEmpty(Parameters.LongRunningOboCacheKey) ? ApiEvent.ApiIds.AcquireTokenOnBehalfOf : ApiEvent.ApiIds.AcquireTokenInLongRunningObo;
```

Comment:
Did we want to identify the difference between initiate and AcquireTokenInLongRunningObo? We were talking about determining who is calling Initiate over and over again in telemetry right?

Reply:
Yea, makes sense, will update.
Currently we can figure out the difference because AcquireTokenInLrObo will use a refresh grant, while InitiateLrObo will use obo grant. But once we update the Initiate method to check the cached assertion, this won't work, since initiate will be able to do refresh grant also.